target/arm/translate-a64.c

```diff
@@ -1448,9 +1486,28 @@ static void disas_cond_b_imm(DisasContext *s, uint32_t insn)
         arm_gen_test_cc(cond, label_match);
         gen_goto_tb(s, 0, s->pc);
         gen_set_label(label_match);
+
+#ifdef CONFIG_FLEXUS
+        FLEXUS_IF_IN_SIMULATION( gen_helper_flexus_insn_fetch_aa64( cpu_env,
+                                        tcg_const_tl(flexus_ins_pc),
```

Comment:
Same here. I won't spam comment anymore, but i think w can solve this pretty quickly with a test